msteams-platform/graph-api/rsc/test-resource-specific-consent.md

```diff
@@ -15,19 +15,195 @@ keywords: teams authorization OAuth SSO Azure AD rsc Postman Graph
 
 Resource-specific consent (RSC) is a Microsoft Teams and Graph API integration that enables your app to use API endpoints to manage specific resources—either teams or chats—within an organization. For more information, see [Resource-specific consent (RSC) — Microsoft Teams Graph API](resource-specific-consent.md).
 
+## Prerequisites
+
+Ensure that you verify the following app manifest changes for resource-specific consent before testing:
+
+<br>
+
+<details>
+
+<summary><b>RSC permissions for app manifest version 1.12</b></summary>
+
```

Comment:
You are probably missing this text here:
Add a [webApplicationInfo](../../resources/schema/manifest-schema.md#webapplicationinfo) key to your app manifest with the following values: